justfile

```
@@ -156,3 +156,7 @@ docs-build-all:
     just docs-apigen --verbose
     just docs-render
     just checklinks docs/_output --offline --no-progress
+
+# open chat
+chat:
+    zulip-term --color-depth 24bit -c ./.zuliprc
```

```suggestion
chat *args:
    zulip-term {{ args }}
```

Not a huge fan of overriding the default behavior of the program, especially since the config file can contain plaintext secrets and will be placed in the current directory if it doesn't exist 😬 .

You can also set all the other settings including color depth in the config file, which IMO obviates the need for passing any arguments here.